lib/rspecq/worker.rb

```diff
@@ -107,7 +108,13 @@ def try_publish_queue!(queue)
       end.map(&:first) & files_to_run
 
       if slow_files.any?
-        puts "Slow files (threshold=#{file_split_threshold}): #{slow_files}"
+        log_event(
+          "Slow files detected (threshold=#{file_split_threshold}): #{slow_files}",
+          "info",
+          slow_files: slow_files,
+          slow_files_count: slow_files.count,
+          file_split_threshold: file_split_threshold
```

Comment:
Isn't this already included in the default extras?

Reply:
Nice catch! Removing it.